src/omero_cli_render.py

```diff
@@ -68,17 +70,17 @@
 
     # Omitted fields will keep their current values, omitted channel-indices
     # will be turned off.
-    bin/omero render edit --copy Screen:1 <YAML or JSON file>
-    # Optimised for bulk-rendering, edits the first image and copies the
-    # rendering settings to the rest. Note using this flag may have different
-    # results from not using it if the images had different settings to begin
-    # with and you are only overridding a subset of the settings (all images
-    # will end up with the same full rendering settings)
-    bin/omero render edit --skipthumbs ...
-    # Update rendering settings but don't regenerate thumbnails
-
-    # %(LIST)s
-    bin/omero render list Image:456
+
+    bin/omero render set --copy Screen:1 <YAML or JSON file>
+    # Equivalent to calling set for a single image followed by calling copy
+    # for all the remaining images in the collection, using the first image
+    # as the source. Note using this flag may have different results from not
```

Comment:
This paragraph is still quite weak. The ``Note`` is trying to explain two cases in one breath, and fails on both imho. 
I think we need an example for 

 bin/omero render set Screen:1 <YAML or JSON file>

which hopefully does what the sentence starting with "Equivalent..." is describing. 
Further, we need another example, which goes

 bin/omero render set --copy Screen:1 <YAML or JSON file>

and the scary "Note" can be simplified there and give a targeted explanation.

Reply:
cc @joshmoore